src/stp/new_plays/GenericPass.cpp

```diff
@@ -41,21 +41,21 @@ GenericPass::GenericPass() : Play() {
                                                                                  std::make_unique<role::Halt>(role::Halt("halt_7"))};
 }
 
-uint8_t GenericPass::score(world_new::World* world) noexcept { return 50; }
+uint8_t GenericPass::score(world_new::World* world) noexcept { return 10; }
 
 void GenericPass::calculateInfoForRoles() noexcept {
     // Keeper
     stpInfos["keeper"].setPositionToShootAt(Vector2{0.0, 0.0});
     stpInfos["keeper"].setEnemyRobot(world->getWorld()->getRobotClosestToBall(world_new::them));
 
-    const Vector2 passingPosition = calculatePassLocation();
+    const Vector2 passingPosition = Vector2();///*Vector2(-field.getFieldLength()/2, -field.getFieldWidth()/2);//*/calculatePassLocation();
```

Comment:
Do not create something that does not exist yet. Use optionals or pointers instead if you want to define a variable at a later time.

Reply:
Why would it not exist? Vector2 is default initialized to (0,0) so it is a completely valid thing to do